xml/installation-installation-configure_lbaas.xml

```diff
@@ -724,338 +606,413 @@ rtt min/avg/max/mdev = 3.473/3.473/3.473/0.000 ms</screen>
-<screen>&prompt.ardana;curl 10.247.96.26
- Welcome to 10.247.94.132</screen>
+     <para>
+      Remove any old FIPS from known_hosts
+     </para>
+<screen><?dbsuse-fo font-size="0.70em"?>&prompt.ardana;ssh-keygen -R  $LB_VM1_FIP
+&prompt.ardana;ssh-keygen -R  $LB_VM2_FIP</screen>
    </step>
    <step>
     <para>
-     Deploy and start the web server on the second virtual machine as you did
-     in the previous steps. Once the second web server is running, list the
-     floating IPs.
+     Deploy the web server app
     </para>
-<screen>&prompt.ardana;openstack floating ip list
-+----------------+------------------+---------------------+---------------+
-| id             | fixed_ip_address | floating_ip_address | port_id       |
-+----------------+------------------+---------------------+---------------+
-| 3ce60...bf55ef | 10.247.94.132    | 10.247.96.26        | 7e5e0...6450e |
-| 680c0...247444 | 10.247.94.133    | 10.247.96.27        | ca95c...36854 |
-| 9a362...b2b22c | 10.247.94.134    | 10.247.96.28        | da28a...c3c51 |
-+----------------+------------------+---------------------+---------------+</screen>
+<screen><?dbsuse-fo font-size="0.70em"?>&prompt.ardana;
+scp -o StrictHostKeyChecking=no -i lb_kp1.pem webserver.sh cirros@$LB_VM1_FIP:webserver.sh 
+ssh -o StrictHostKeyChecking=no -i lb_kp1.pem cirros@$LB_VM1_FIP 'chmod +x ./webserver.sh'
+ssh -o StrictHostKeyChecking=no -i lb_kp1.pem cirros@$LB_VM1_FIP '(./webserver.sh&amp;)'
+ 
+scp -o StrictHostKeyChecking=no -i lb_kp1.pem webserver.sh cirros@$LB_VM2_FIP:webserver.sh 
+ssh -o StrictHostKeyChecking=no -i lb_kp1.pem cirros@$LB_VM2_FIP 'chmod +x ./webserver.sh'
+ssh -o StrictHostKeyChecking=no -i lb_kp1.pem cirros@$LB_VM2_FIP '(./webserver.sh&amp;)'</screen>
    </step>
    <step>
     <para>
-     Display the floating IP for the load balancer.
+     Make sure the webservers respond with their IPs
     </para>
-<screen>&prompt.ardana;openstack floating ip show 9a3629bd-b0a6-474c-abe9-89c6ecb2b22c
-+---------------------+--------------------------------------+
-| Field               | Value                                |
-+---------------------+--------------------------------------+
-| fixed_ip_address    | 10.247.94.134                        |
-| floating_ip_address | 10.247.96.28                         |
-| floating_network_id | d3cb12a6-a000-4e3e-82c4-ee04aa169291 |
-| id                  | 9a3629bd-b0a6-474c-abe9-89c6ecb2b22c |
-| port_id             | da28aed3-0eb4-4139-afcf-2d8fd3fc3c51 |
-| router_id           | 6aafc9a9-93f6-4d7e-94f2-3068b034b823 |
-| status              | ACTIVE                               |
-| tenant_id           | 4b31d0508f83437e83d8f4d520cda22f     |
-+---------------------+--------------------------------------+</screen>
+<screen>&prompt.ardana;curl $LB_VM1_FIP
+Welcome to 12.12.12.5
+curl $LB_VM2_FIP
+Welcome to 12.12.12.5</screen>
    </step>
    <step>
     <para>
-     Test the load balancing.
+     Access the wevservers thru the LB
```

Comment:
```suggestion
     Access the wevservers through the loadbalancer:

Reply:
Thru?!?!?!?!

Reply:
_wevservers_?!

Reply:
wevservers was clearly a typo ;-)  ... updated thru to through... I should've proofread a little more carefully